src/main/flatbuffers/rlbot.fbs

```diff
@@ -158,6 +165,31 @@ table GameTickPacket {
 
 root_type GameTickPacket;
 
+/// The state of a rigid body in RL's physics engine.
+table RBState {
+  frame:int;
+  quaternion:Quaternion;
+  location:Vector3;
+  velocity:Vector3;
+  angularVelocity:Vector3;
+}
+
+table PlayerPhysicsInfo {
```

Comment:
I feel like making this "RawPlayerPhysicsInfo" to make it stand out.

Reply:
I'm conflicted about the name, but some `///` documentation definitely wouldn't go amiss (which gets translated to javadoc, etc when classes are generated).

Reply:
Can do comments.

I think this is more deserving of the name "Physics" than the others, but we can't go back and change the old names of course.

I don't care about the shed color, someone decree a list of names and I'll change them to whatever